Dockerfile

```diff
@@ -44,13 +56,10 @@ ENV CI_SMOKE_TEST=true
 
 CMD ["npm", "run", "start:prod"]
 
-# docker build --target main -t scheduled --build-arg BUILD_MODE=ci .
-# if you are on Mac M1 please add --platform=linux/amd64 after FROM below
-FROM node:14-alpine AS scheduled
-WORKDIR /usr/src/scheduler
-COPY --from=main /usr/src/app/dist ./dist/
-COPY --from=main /usr/src/app/node_modules ./node_modules/
-COPY --from=main /usr/src/app/docker/apply/forms-json ./docker/apply/forms-json/
-COPY --from=main /usr/src/app/src/server/models/db/schema.prisma ./src/server/models/db/
-COPY --from=main /usr/src/app/src/server/models/db/migrations ./src/server/models/db/migrations
+# docker build --target scheduled -t scheduled --build-arg BUILD_MODE=ci .
+FROM prod AS scheduled
+USER 1001
```

Comment:
Is USER 1001 a linux thing?

Reply:
`USER` is a docker thing "do the next steps as user with id 1001". There are user and user groups in Mac and windows as well (but I don't know how/if they work on a windows/Mac base image). user 1001 is also named appuser (in the appuser group. see lines 3 and 4).